recipes/openmm-cpu/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "openmm" %}
+{% set version = "7.3.1" %}
+{% set checksum = "db0c1fddc3068f689931385bc4009529261300a19effc2a9d3d47e43ab752875" %}
+
+package:
+  name: openmm-cpu
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: False
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - jom  # [win]
+
+  host:
+    - python  {{ python }}
+    - fftw
+    - swig
+    - doxygen ==1.8.14
```

```suggestion
    - doxygen 1.8.14
```
I think you can drop the docs build, or do you really need want/need them?

---

I'll check with the authors whether they want them bundled.